src/Atmos/Parameterizations/CloudPhysics/Microphysics.jl

```diff
@@ -335,22 +337,24 @@ a constant timescale.
 function conv_q_vap_to_q_liq_ice(
     liquid_param_set::ALPS,
     q_sat::PhasePartition{FT},
-    q::PhasePartition{FT},
+    q::PhasePartition{FT};
+    slowdown_liq::FT = FT(1),
 ) where {FT <: Real}
 
     _τ_cond_evap::FT = τ_relax(liquid_param_set)
 
-    return (q_sat.liq - q.liq) / _τ_cond_evap
+    return (q_sat.liq - max(FT(0), q.liq)) / _τ_cond_evap / slowdown_liq
```

Comment:
```suggestion
    return (q_sat.liq - q.liq) / _τ_cond_evap / slowdown_liq
```
We can filter `q.liq` on the construction of `PhasePartition`